utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -178,19 +178,39 @@ def _cleanup_by_path(self, path):
 
 
 class OscapScan(object):
-    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=os.getcwd(),
+    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=None,
                  hours_old=2):
         self.tmp_dir = tmp_dir
         self.helper = OscapHelpers(tmp_dir)
         self.mnt_dir = mnt_dir
         self.hours_old = hours_old
 
+    def _get_mnt_dir(self):
+        '''
+        Get directory for mount container
+        '''
+        if self.mnt_dir is None:
+            return tempfile.mkdtemp()
+        else:
+            return self.mnt_dir
+
+    def _remove_mnt_dir(self, mnt_dir):
+        '''
+        Remove temporary directory if it was not passed by __init__
+        We don't want to remove directory what was created by used
+        '''
+        if self.mnt_dir != mnt_dir:
```

Comment:
If I should do more refactoring, I would
- merge common code of scan/scan_cve - maybe use __enter__/__exit__
- remove tmp_dir and mnt_dir from __init__ signature - it will break some use case, but as I know, nowhere it is used now.

(there is still some quite confusing parts like "_cleanup_by_path" Create new DockerMount instance)

Please, share your expectation.